plugins/geospatial/server/geospatial.py

```diff
@@ -492,10 +492,10 @@ def setGeospatial(self, item, params):
         .param('id', 'The ID of the item.', paramType='path')
         .param('body', 'A JSON object containing the geospatial fields to add.',
                paramType='body')
-        .errorResponse('ID was invalid.')
-        .errorResponse('Invalid JSON was passed in request body.')
-        .errorResponse('Geospatial key name was invalid.')
-        .errorResponse('Geospatial field did not contain valid GeoJSON.')
+        .errorResponse('ID was invalid.\n\n'
+                       'Invalid JSON was passed in request body.\n\n'
+                       'Geospatial key name was invalid.\n\n'
+                       'Geospatial field did not contain valid GeoJSON.')
```

Comment:
(I'm assuming the `\n\n` is a swagger detail, which would be best not to leak, but even if not, we are really trying to represent a list here, so it makes sense for our API to accept one and we can internally serialize it however we need to.)